docs/mindsdb-docs/docs/sql/tutorials/house-sales-forecasting.md

```diff
@@ -1,60 +1,158 @@
+    A **feature** is an input variable — the x variable in simple linear regression.
+
+## Training a Predictor Via [`#!sql CREATE PREDICTOR`](/sql/create/predictor)
+
+Let's create and train your first machine learning predictor. For that, we are going to use the [`#!sql CREATE PREDICTOR`](/sql/create/predictor) syntax where we specify what sub-query to train `#!sql FROM` (features) and what we want to `#!sql PREDICT` (labels).
+
+```sql
+CREATE PREDICTOR mindsdb.house_sales_predictor
 FROM files
-  (SELECT * FROM HR_MA)
+  (SELECT * FROM house_sales)
 PREDICT MA
 ORDER BY saledate
 GROUP BY bedrooms, type
 -- as the target is quarterly, we will look back two years to forecast the next one
 WINDOW 8
-HORIZON 4;  
+HORIZON 4;
 ```
 
-You can check the status of the predictor:
+We use all of the columns as features, except for the `MA` column whose value is going to be predicted.
 
+MindsDB makes it simple so that we don't need to repeat the predictor creation process for every group, that is, for every number of bedrooms or for every type of a real estate. Instead, we just group by both the `bedrooms` and `type` columns and the predictor learns from all series and enables forecasts for all of them!
+
+## Checking the Status of a Predictor
+
+A predictor may take a couple of minutes for the training to complete. You can monitor the status of your predictor by using this SQL command:
+
+```sql
+SELECT status
+FROM mindsdb.predictors
+WHERE name='house_sales_predictor';
 ```
-SELECT * FROM mindsdb.predictors where name='home_sales_model';
-```
 
-## Generating some forecasts
+If we run it right after creating a predictor, we'll most probably get this output:
+
+```sql
++----------+
+| status   |
++----------+
+| training |
++----------+
+```
 
-Once the predictor has been successfully trained, you can query it to get forecasts for a given period of time. Usually, you'll want to know what happens right after the latest training data point that was fed, for which we have a special bit of syntax, the "LATEST" key word:
+But if we wait a couple of minutes, this should be the output:
 
+```sql
++----------+
+| status   |
++----------+
+| complete |
++----------+
 ```
-SELECT m.saledate as date,
-       m.MA as forecast
-FROM mindsdb.home_sales_model as m 
-JOIN files.HR_MA as t
+
+Now, if the status of our predictor says `complete`, we can start making predictions!
+
+## Making Predictions
+
+You can make predictions by querying the predictor as if it were a table. The [`SELECT`](/sql/api/select/) syntax lets you make predictions for the label based on the chosen features for a given period of time. Usually, you want to know what happens right after the latest training data point that was fed. We have a special keyword for that: the `LATEST` keyword.
+
+```sql
+SELECT m.saledate as date, m.MA as forecast
+FROM mindsdb.house_sales_predictor as m 
+JOIN files.house_sales as t
 WHERE t.saledate > LATEST AND t.type = 'house' AND t.bedrooms = 2
```

Comment:
Break the `AND`s on multiple lines to maximize readability.

Reply:
Done.